bin/start_fe.sh

```diff
@@ -127,21 +127,7 @@ fi
 #
 # NOTE: try not adding new JAVA_OPTS_FOR_JDK_## for a new JDK version.
 #
-final_java_opt=
-case $JAVA_VERSION in
-  11)
-    # take JAVA_OPTS_FOR_JDK_11 or JAVA_OPTS_FOR_JDK_9 if the former is empty
-    final_java_opt=${JAVA_OPTS_FOR_JDK_11:-$JAVA_OPTS_FOR_JDK_9}
-    final_java_opt=${final_java_opt:-$JAVA_OPTS}
-    ;;
-  9)
-    # take JAVA_OPTS_FOR_JDK_9 or JAVA_OPTS if the former is empty
-    final_java_opt=${JAVA_OPTS_FOR_JDK_9:-$JAVA_OPTS}
-    ;;
-  *)
-    final_java_opt=$JAVA_OPTS
-    ;;
-esac
+final_java_opt=$JAVA_OPTS
```

Comment:
since it is supposed to upgrade from jdk11 to jdk17, may need to consider support JAVA_OPTS_FOR_JDK_11 opts as well, avoid sudden java opts reset within the upgrade.